### PR TITLE
Fix avatar fade

### PR DESCRIPTION
**Pull request by sethalves**
- avatars once again fade (shrink) when they leave the domain
- the rate at which an avatar fades has been increased

### CODE REVIEW

**Comment by sethalves** [comment]
https://app.asana.com/0/26485730907942/91754833406689

**Comment by sethalves** [comment]
to test:
Log  two avatars in so they can see each other.  Log one out and watch it shrink.

**Comment by hifi-gustavo** [comment]
Build console output for commit: ee5a82f9b80af83fd3e454cd43892958d6e21b8f available at: https://gist.github.com/90fe23c80a01a1a5c97b<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7141.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7141.dmg

**Comment by sethalves** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: ee5a82f9b80af83fd3e454cd43892958d6e21b8f available at: https://gist.github.com/90fe23c80a01a1a5c97b<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7141.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7141.dmg

**Comment by sethalves** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: ee5a82f9b80af83fd3e454cd43892958d6e21b8f available at: https://gist.github.com/90fe23c80a01a1a5c97b<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7141.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7141.dmg

File: libraries/render-utils/src/Model.cpp [hunk 1]
```diff
@@ -92,7 +92,7 @@ void Model::setScale(const glm::vec3& scale) {
     _scaledToFit = false;
 }
 
-const float METERS_PER_MILLIMETER = 0.01f; 
+const float METERS_PER_MILLIMETER = 0.01f;
```
Comment (AndrewMeadows) [inline libraries/render-utils/src/Model.cpp]: I know you didn't change the value of this but... this numerical value is too big by a factor of 10.

File: libraries/render-utils/src/Model.cpp [hunk 2]
```diff
@@ -100,7 +100,7 @@ void Model::setScaleInternal(const glm::vec3& scale) {
         if (_scale.x == 0.0f || _scale.y == 0.0f || _scale.z == 0.0f) {
             assert(false);
         }
-        initJointTransforms();
+        simulate(0.0f, true);
```
Comment (AndrewMeadows) [inline libraries/render-utils/src/Model.cpp]: This must be the actual fix.  The rest of the changes don't actually change much.

**Comment by hifi-gustavo** [comment]
Build console output for commit: fea60499d44008df457fb44cf30fcc6787ad7f8b available at: https://gist.github.com/90fe23c80a01a1a5c97b<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7141.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7141.dmg

**Pull request merged**